core/app_switcher/app_switcher.py

```diff
@@ -23,6 +25,9 @@
 # a list of the current overrides
 overrides = {}
 
+# apps to exclude from running list
+excludes = set()
```

Comment:
Probably worth adding a comment that any line without a comma is excluded? This is a bit different from several of the other CSVs, where it's assumed the spoken form and result are the same in this case.

Reply:
Agreed, that was a bit too mysterious (and comments in this file were out of date). Updated.